HW02/ihar_shkuryn/hw02.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module MyArrayMethods
+  refine Array do
+    def my_map
+      if block_given?
+        results = []
+        i = 0
```

Comment:
try to avoid naming like `I`, `x` and etc.
You should use more informative variables naming

Reply:
done